Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Credentials
+    {
+        private const string Separator = ",";
+
+        public string CredentialsFile { get; } = "credentials.db";
```

Comment:
would anything outside of this class need to know the file location?

Reply:
Maybe to display it while working with it? I changed it to private for now.